### DISCUSSION TITLE
Looking for an Ethernet Thingino compatible camera

### DISCUSSION THREAD

**Discussion by fmnamado** [themactep/thingino-firmware#242] [General]
Hello,

I am in the market looking for an Ethernet Thingino compatible camera.

For what I saw in the supported hardware, all of them seemed like Wifi only.
I leave the suggestion of adding the information explicitely (LAN Yes/No, Wi-Fi Yes/No).

Does an Ethernet Thingino compatible exists?

Thank you.

**Comment by themactep** [comment]
All Vanhua modules use Etherent network, some WiFi cameras have two interfaces - wireless and wired.

**Comment by fmnamado** [comment]
I am sorry, i wasn't clear then. I want a camera with wired interface. The wireless for me is an optional.

Which models have wired interface?

Thank you.

**Comment by themactep** [comment]
From what I remember, Feisda WF-HD620 has Ethernet port. You will need to make your own research to see which other models do. Use Google and check cameras configs for clues.
